src/main/java/net/snowflake/ingest/streaming/internal/RowBufferStats.java

```diff
@@ -95,13 +129,25 @@ static RowBufferStats getCombinedStats(RowBufferStats left, RowBufferStats right
               "left=%s, right=%s",
               left.getCollationDefinitionString(), right.getCollationDefinitionString()));
     }
+
+    if (left.enableDistinctValuesCount != right.enableDistinctValuesCount) {
```

Comment:
need similar chekc for enableValuesCount, no?

Reply:
Added check.